src/latticpp/ckks/ciphertext.h

```diff
@@ -12,6 +12,8 @@ namespace latticpp {
 
     double scale(const Ciphertext &ct);
 
+    int degree(const Ciphertext &ct);
```

Comment:
Make this return a `uin64_t`, or add an explicit cast here from the `uint64_t` returned from Go (see comment above). I want it to be very obvious that the integer types align across language boundaries.